core/modules/system/system.install

```diff
@@ -3274,6 +3274,33 @@ function system_update_1081() {
   $config->save();
 }
 
+/**
+ * Remove superfluous test files if they are present.
+ */
+function system_update_1082() {
+  // Crufty files that may have been left behind after PR 3911.
+  // @see https://github.com/backdrop/backdrop-issues/issues/5465
+  $base = BACKDROP_ROOT . '/core/modules/system/tests/';
+  $filenames = array(
+    $base . 'cron_queue_test.module',
+    $base . 'system_cron_test.module',
+  );
+  $removed = array();
+  foreach ($filenames as $filename) {
+    if (file_exists($filename)) {
+      if (unlink($filename)) {
+        $removed[] = $filename;
+      }
+      else {
+        watchdog('system', t('Unable to remove file %filename', array('filename' => $filename)), WATCHDOG_ERROR);
```

Comment:
watchdog() in an update hook is very uncommon.

I think if something went wrong in an update hook, it's supposed to be returned directly via backdrop_set_message() and the appropriate type (warning, error).

Reply:
Thx, updated.